lib/src/configs/cosim_wrap_config.dart

```diff
@@ -124,9 +143,61 @@ class CosimWrapConfig extends CosimProcessConfig {
           '-f',
           _cosimMakefileName,
         ],
-        environment: environment,
+        environment: {
+          if (environment != null) ...environment!,
+          'PATH':
```

Comment:
I think you only want to update the path and python3 if `usePythonVirtualEnvironment` is true?